_templates/component/new/stories.ejs.t

```diff
@@ -0,0 +1,21 @@
+---
+to: src/components/<%= name %>/<%= name %>.stories.tsx
+---
+import decoratorCentered from "@storybook/addon-centered"
+import { <%= name %> } from "./<%= name %>"
+
+/**
+ * See Storybook Docs: Writing Stories
+ * https://storybook.js.org/docs/basics/writing-stories/
+ */
+
+export default {
+  title: "<%= name %>",
+  decorators: [decoratorCentered],
```

Comment:
This addon might now be obsolete, there's a "native" way of setting that:
```tsx
layout: 'centered',
```

But we probably need to revisit/update storybook in a new PR

Reply:
Yep, I imagine there might be several updates we need to make regarding Storybook 👍